scale-tests/setup_streaming.py

```diff
@@ -153,13 +153,16 @@ def get_expected_task_count(service_options: dict) -> int:
                                         zookeeper_service["service"]["name"],
                                         "endpoint clientport", json=True)["dns"]
 
+        expected_count = get_expected_task_count
```

Comment:
Why do we need the intermediate variable? We evaluate the function in each loop iteration, so there should not be any scope issues.

Reply:
Sorry @porridge this should have been `expected_count = get_expected_task_count()` so the function was evaluated. I'll fix